roles/rsyslog/README.md

```diff
@@ -51,7 +51,8 @@ Currently, the role supports 3 types of logs collections ([input_roles](https://
 
 To make an effect with the following setting, vars.yaml has to have `rsyslog_enabled: true`.  Unless `rsyslog_enabled` is set to true, LSR/Logging does not deploy rsyslog config files.
 ```
-rsyslog_enabled: true
+rsyslog_enabled: True
```

Comment:
just a nit, but why are you changing true to True? Ansible documentation almost universally uses yes/no or true/false (lowercase).

Reply:
Oops.  Sorry...  Do you recommend to consolidate all the boolean setting to yes/no?  (I think I read it in some doc...)

Reply:
Maybe, but it wouldn't belong to this PR anyway (such stylistic changes have no relation with the subject of the PR, which is my other objection to this particular change).

Reply:
@pcahyna, I've opened another PR [1] for these issues.  
- Replacing 'yum' with 'package'.
- Removing quotes from single values in the tasks.
- Lowering the letter cases of True/False.

Could you please take a look?  Thanks!
[1] https://github.com/linux-system-roles/logging/pull/85

Reply:
> just a nit, but why are you changing true to True? Ansible documentation almost universally uses yes/no or true/false (lowercase).

@pcahyna, undid this change.  (sorry for my slow understanding...)  Please let me know if there are any other changes I should eliminate.  Thanks!